### PR TITLE
Further Updates to the English Heritage Landing Page

### PR DESCRIPTION
**Pull request by AWare**
Still need new hero image.

Fixes: 
image

image

Update to English Heritage logo
image

### CODE REVIEW

File: frontend/app/configuration/Config.scala [hunk 1]
```diff
@@ -159,11 +159,11 @@ object Config {
       codes = PromoCodeSet(PromoCode("EH2016")),
       description = "Free English Heritage membership worth £88 when you become a Partner or Patron Member",
       expires = DateTime.parse("2016-04-01T01:00:00Z"),
-      imageUrl = "http://lorempixel.com/400/200/abstract",
+      imageUrl = "https://s3-eu-west-1.amazonaws.com/memsub-promo-images/eh2016.png",
```
Comment (joelochlann) [inline frontend/app/configuration/Config.scala]: Think this approach might require a bit more thought - perhaps a chat with another frontend developer

Comment (AWare) [inline frontend/app/configuration/Config.scala]: Quick update: Have had chat, seems dotcom use their own bucket for things like this- so not unprecedented.

File: frontend/app/views/fragments/page/sectionCardStack.scala.html [hunk 2]
```diff
@@ -2,9 +2,10 @@
 
 <section class="page-slice page-slice--listing">
     <div class="l-constrained">
+        @if(!leadIn.isEmpty){<div class="listing__lead-in listing__lead-in--raised u-hidden-from-mem-full"><h3 class="h-intro">@leadIn.get</h3></div>}
```
Comment (joelochlann) [inline frontend/app/views/fragments/page/sectionCardStack.scala.html]: could do this as a for comprehension

```html
@for(leadIn <- leadIn) {
  <div class="listing__lead-in listing__lead-in--raised u-hidden-from-mem-full">
    <h3 class="h-intro">@leadIn</h3>
  </div>
}
```

File: frontend/app/views/fragments/page/sectionCardStack.scala.html [hunk 3]
```diff
@@ -2,9 +2,10 @@
...
         <div class="listing listing--narrow listing--card-stack @if(leadIn.isEmpty){listing--empty-lead-in}">
             @* If lead-in is empty we still need its containing div for column alignment at larger widths *@
-            <div class="listing__lead-in"><h3 class="h-intro">@leadIn.getOrElse("")</h3></div>
+            <div class="listing__lead-in u-hidden-until-mem-full">@if(!leadIn.isEmpty){<h3 class="h-intro">@leadIn.get</h3>}</div>
```
Comment (joelochlann) [inline frontend/app/views/fragments/page/sectionCardStack.scala.html]: this line and the one above is long and hard to read - please add some newlines & indentation to make it clearer

**Comment by joelochlann** [comment]
:+1: except for the comment about long lines

File: frontend/app/configuration/Config.scala (hunk 1 above)
Comment (joelochlann) [inline frontend/app/configuration/Config.scala]: :+1:

**Comment by joelochlann** [comment]
:+1: looks good

**Pull request merged**